sass/style-short.scss

```diff
@@ -1,5 +1,6 @@
 #logo {
-  height: 100px;
+  background-image: url(../css/images/gravity.coast.cassette.web.short.png);
```

Comment:
The css "src" is ./css

This folder is copied to dist in the gulp copy task.

The sass files are in ,/scss they get compiled and dropped in dist/css

So after gulp all css and compiled sass is in dist/css

You are viewing dist/index.html which includes css/style.css (under
dist, the compiled sass)

The references just need to resolve in the browser, when you load it.

On Tuesday, April 21, 2015, sundialmusic <notifications@github.com> wrote:

> In sass/style-short.scss
> <https://github.com/sundialmusic/gravitycoast/pull/7#discussion_r28804954>
> :
>
> > @@ -1,5 +1,6 @@
> >  #logo {
> > -  height: 100px;
> > +  background-image: url(../css/images/gravity.coast.cassette.web.short.png);
>
> There is an image folder in dist/css/ ... should that not be copied? Not
> be used? Also, since gulp is creating the dist folder, where should I
> actually add the resource to the source? That's the part that's confusing:
> I add a .png to the SOURCE (not the dist)... so at SOME POINT, don't I have
> to tell gulp where it actually is before Gulp can gulp it up and put it
> where it wants to? Should I create a variable for the image in the mixin? I
> thought the point of the Dist / Gulp thing was to let Gulp rearrange the
> file names, file structures etc. in order to obscure the code and make it
> more efficient. How can I link to something that hasn't been generated yet?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sundialmusic/gravitycoast/pull/7/files#r28804954>.
>